README.rst

```diff
@@ -173,6 +173,24 @@ This configuration allows you to hit Tab to get auto-complete, and to call
 various functions to rename and get definitions. Of course you are free to
 configure this any way you'd like.
 
+Emacs
+-------
+
+With lsp-mode and use-package:
+
+.. code:: emacs-lisp
+
+   (use-package nim-mode
+     :ensure t
+     :hook
+     (nim-mode . lsp)
+     :commands nim-mode
```

Comment:
You don't need this line, since if you have `nim-mode` installed, the mode is automatically enabled for the correct file extension (see https://github.com/nim-lang/nim-mode/blob/16a245e4974d21a6e9e7163e7fbfb50a0bd89f99/nim-mode.el#L238). For the same reason, you don't need `:ensure t`; as soon as a file with the correct extension is opened, the mode will be lazily loaded.